Bugfix/566 creation date time format inconsistent

ywkim312 opened the pull request.

sandeep-ps reviewed (commented):
Please fix the merge conflict in CHANGELOG and move the change log entry to [Unreleased] -> Changed section at the top. Rest looks good. Thanks.

ywkim312 commented:
> Please fix the merge conflict in CHANGELOG and move the change log entry to [Unreleased] -> Changed section at the top. Rest looks good. Thanks.

Fixed and moved
